interface/usergroup/user_info.php

```diff
@@ -127,6 +128,9 @@ function(data)
                         <label class='control-label col-sm-2'><?php echo xlt('New Password') . ":"; ?></label>
                         <div class='col-sm-3'>
                             <input type='password' class='form-control' name='newPass'  value="" autocomplete='off'>
+                            <!-- Password Strength Meter -->
+                            <div id="password_strength_meter" style="width:100%; height:15px; border:1px solid #ccc; margin-top: 5px;"></div>
```

Comment:
don't use embedded styling. instead use bootstrap classes for styling (see this scripts and other scripts for examples)

Reply:
Looks like this is resolved.